packages/shared/src/errors/Error.ts

```diff
@@ -69,6 +90,58 @@ export class ClerkAPIResponseError extends Error {
   };
 }
 
+/**
+ * Custom error class for representing Clerk runtime errors.
+ *
+ * @class ClerkRuntimeError
+ * @example
+ *   throw new ClerkRuntimeError('An error occurred', { code: 'password_invalid' });
+ */
+export class ClerkRuntimeError extends Error {
+  /**
+   * Indicates that this is a Clerk runtime error.
+   *
+   * @type {boolean}
+   * @memberof ClerkRuntimeError
+   */
+  clerkRuntimeError: true;
```

Comment:
❓ should we use the `clerkError` instead? Also since `isClerkRuntimeError` helper exists, do we need this?

Reply:
I removed this one

Reply:
i know. i'm saying that instead of removing it, using it here.